tools/sign_executable/sign_executable.py

```diff
@@ -87,6 +87,7 @@
         "ditto",
         "-c",
         "-k",
+        "--sequesterRsrc",
```

Comment:
There is a comment above this line of code that explains what all the arguments do. Please update the comment to explain what this new arg is doing.